docs/api.md

```diff
@@ -43,6 +43,7 @@ Here major and breaking changes to the API are listed by version.
 - [GET /projects/:id/datasets](#reference/datasets/datasets/datasets) now supports `X-Extended-Metadata` header to retrieve number of Entities in the Dataset and timestamp of the last Entity
 - `$select` in OData now supports selecting complex type(groups) 
 - [Creating a form](#reference/forms/forms/creating-a-new-form) can now return workflow warnings
+- The response of `GET`, `POST`, `PUT` and `PATCH` methods of [Submissions](#reference/submissions/listing-all-submissions-on-a-form) endpoint has been updated to include metadata of the `currentVersion` of the Submission. 
```

Comment:
This is in the changelog for .1. What do you think about creating a new section for .2?